dash/dash-config.cmake.in

```diff
@@ -8,14 +8,14 @@
 
 @PACKAGE_INIT@
 
-set(DASH_VERSION_MAJOR "@DASH_VERSION_MAJOR@")
-set(DASH_VERSION_MINOR "@DASH_VERSION_MINOR@")
-set(DASH_VERSION_PATCH "@DASH_VERSION_PATCH@")
+set(DASH_VERSION_MAJOR "@DASH_VERSION_MAJOR@")
+set(DASH_VERSION_MINOR "@DASH_VERSION_MINOR@")
+set(DASH_VERSION_PATCH "@DASH_VERSION_PATCH@")
 
 set(DASH_LIBRARY "@DASH_LIBRARY@")
 set(DASH_LIBRARIES ${DASH_LIBRARY} "dart-@dart_variant@")
 set(DASH_CXX_FLAGS "@VARIANT_ADDITIONAL_COMPILE_FLAGS@ @CMAKE_CXX_FLAGS_RELEASE@")
```

Comment:
Currently we always export the release target and there is no possibility to install a debug build of DASH. IMO we should export the current target. I already fixed this [here](https://github.com/dash-project/dash/blob/feat-coarray/dash/dash-config.cmake.in#L17). However this is still not optimal as some flags (e.g. OpenMP) are still not exported.

While you are on this, can you provide a fix?